test/specs/lib.environment.spec.js

```diff
@@ -0,0 +1,19 @@
+const { hasStore } = require('../../server/lib/environment.js');
+
+describe('hasStore', () => {
+  
+  test(`returns false when the DAV_ENV environment variable isn't set to 'simulated'`, () => {
+    expect.assertions(1);
+    if (process.env.DAV_ENV !== 'simulation') {
+      expect(hasStore).toBe(false);
+    }
+  });
+
+  test(`returns true when the DAV_ENV environment variable is set to 'simulated'`, () => {
```

Comment:
We can remove this test. It won't even run in a testing environment because the `if` condition will return false.

If in the future we change how we handle environment in a testing environment, we can add something here.